src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -128,6 +128,7 @@ object JavaServerAppPackaging extends AutoPlugin {
         makeStartScriptReplacements,
       linuxScriptReplacements += JavaServerLoaderScript.loaderFunctionsReplacement(serverLoading.value, ARCHETYPE),
       linuxScriptReplacements ++= bashScriptEnvConfigLocation.value.map(ENV_CONFIG_REPLACEMENT -> _).toSeq,
+      linuxScriptReplacements ++= rpmScriptReplacements.value,
```

Comment:
I feel `linuxScriptReplacements in Rpm` is the way most things work in sbt-native-packager.